docs/src/shared/CodeBlockActionButtonRow.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { CopyToClipboard } from './CopyToClipboardButton';
+import { FullCodeOnGithub } from './FullCodeOnGithubButton';
+import { SxProps } from '@mui/material/styles';
+
+export type CodeBlockActionButtonRowProps = {
+    shouldRenderCopyButton?: boolean;
+    shouldRenderGithubButton?: boolean;
```

Comment:
No need to add these props. You can just check if copytext or a url are present. If they are, render the respective button.